west.yml

```diff
@@ -108,7 +108,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 719399b7a839be1318bba7fd66d9bd13323de278
+      revision: d67aa3bfef770208668fc2ccb1c08a50090d3202
```

Comment:
Please rebase. And could you change SHA to `b130fbdbf17d88660b234268c7598607f3ad270c` (most recent commit, I'd like to avoid updating the manifest twice).